evalkit_api_client/evalkit_api_client/types.py

```diff
@@ -0,0 +1,5 @@
+from typing import Any, Dict, Optional
```

Comment:
Ugh with the typing...

Reply:
i'm giving it a try, i want to learn more about what it offers and what its limitations are.

Reply:
yes, it's our interest to keep development consistent here as well as in the `canvas_api_client`. if you think of it as being stuck, i'd encourage you to think of it in another way. it gives us a chance to utilize the benefits of the type system, and learn how to use it.

Reply:
Okay.